webpack.config.js

```diff
@@ -1,2 +1,10 @@
-export * from 'react-project/webpack'
+import webpack from 'webpack'
+import { ServerConfig, ClientConfig } from 'react-project/webpack'
 
+ClientConfig.plugins.push(
+  new webpack.DefinePlugin({
+    'process.env.ACCESS_TOKEN': JSON.stringify(process.env.ACCESS_TOKEN)
```

Comment:
@brad - yeah for some reason it didn't work just as is.  Regardless have used a more appropriate web plugin that does away with it :)